src/main/java/tools/jackson/databind/deser/jdk/JDKKeyDeserializer.java

```diff
@@ -360,13 +360,13 @@ final static class EnumKD extends JDKKeyDeserializer
         protected final AnnotatedMethod _factory;
 
         /**
-         * Alternative to parse enums using {@code toString()} method as the source.
+         * Alternative to parse using {@code toString()} method as the source.
          * Works when {@link DeserializationFeature#READ_ENUMS_USING_TO_STRING} is enabled.
          */
         protected final EnumResolver _byToStringResolver;
 
         /**
-         * Alternative to parse enums using index with {@link Enum#ordinal()} method as the source.
+         * Alternative to parse using index from {@link Enum#ordinal()} method as the source.
```

Comment:
same problem with dropping the object from the clause, this no longer reads well. And I think `with` is better than `from`. For me, this edit is worse than the existing text.